lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -124,25 +124,24 @@
 
 import re
 from time import sleep
+from ansible import constants as C
```

Comment:
Imports from `ansible` are not available to modules. Imports must be from `ansible.module_utils`.